gdx/src/com/badlogic/gdx/graphics/Texture.java

```diff
@@ -124,6 +124,14 @@ public Texture (TextureData data) {
 		load(data);
 		if (data.isManaged()) addManagedTexture(Gdx.app, this);
 	}
+
+	public Texture (TextureData data, int glHandle) {
+		super(GL20.GL_TEXTURE_2D, glHandle);
+		load(data);
+		if (data.isManaged()) {
+			addManagedTexture(Gdx.app, this);

```

Comment:
What'd be a good way to handle this case? Always consider the texture unmanaged and throw an error if data.isManaged() returns true?